service/src/config.rs

```diff
@@ -215,17 +216,43 @@ impl From<&ArgMatches<'_>> for Config {
 #[derive(Clone, Debug, PartialEq, Serialize, Deserialize)]
 pub struct RunConfig {
 	/// Skip remote attestation. Set this flag if running enclave in SW mode
-	pub skip_ra: bool,
+	skip_ra: bool,
 	/// Set this flag if running in development mode to bootstrap enclave account on parentchain via //Alice.
-	pub dev: bool,
+	dev: bool,
 	/// Request key and state provisioning from a peer worker.
-	pub request_state: bool,
+	request_state: bool,
 	/// Shard identifier base58 encoded. Defines the shard that this worker operates on. Default is mrenclave.
-	pub shard: Option<String>,
+	shard: Option<String>,
 	/// Optional teeracle update interval
-	pub teeracle_update_interval: Option<Duration>,
+	teeracle_update_interval: Option<Duration>,
 	/// Marblerun's Prometheus endpoint base URL
-	pub marblerun_base_url: Option<String>,
+	marblerun_base_url: Option<String>,
+}
+
+impl RunConfig {
+	pub fn skip_ra(&self) -> bool {
+		self.skip_ra
+	}
+
+	pub fn dev(&self) -> bool {
+		self.dev
+	}
+
+	pub fn request_state(&self) -> bool {
+		self.request_state
+	}
+
+	pub fn shard(&self) -> Option<&str> {
+		self.shard.as_deref()
+	}
+
+	pub fn teeracle_update_interval(&self) -> Duration {
+		self.teeracle_update_interval.unwrap_or(DEFAULT_MARKET_DATA_UPDATE_INTERVAL)
+	}
+
+	pub fn marblerun_base_url(&self) -> &str {
+		self.marblerun_base_url.as_deref().unwrap_or("http://localhost:9944")
```

Comment:
This was the default that was used, but it does not make sense to me because this is also the default port of a substrate node. @OverOrion is this really the default port that the marblerun uses?

Reply:
It is: https://github.com/edgelesssys/marblerun/blob/master/docs/docs/workflows/monitoring.md?plain=1#L26

Reply:
Alright thanks! added it as a comment: 48c538c380364c20079c3a75b09cf1da7b218f6b